test/test-dir.js

```diff
@@ -20,14 +20,14 @@ describe('dir', function () {
 		w.close();
 	});
 
-	it('should not watch files inside directory', function (done) {
+	it('should watch files inside directory', function (done) {
```

Comment:
NOTE: this is the correct behavior; it aligns with gulp-watch <= 4.3.5 as well as gulp 4's `gulp.watch`.